### PR TITLE
Revisions to "Introducing Visual Open Authoring"

### PR DESCRIPTION
**Pull request by dwalkr**
The main thing I did here was reduce the size of the Next.js section. I don't think we need to go into that much detail (maybe a later post) and it starts to take away from the main point.

aside from that it's mostly smoothing language, enforcing good information hierarchy, and driving home this **Visual Open Authoring** term

### CODE REVIEW

File: content/blog/introducing-visual-open-authoring.md [hunk 1]
```diff
@@ -8,40 +8,34 @@ consumes: null
...
+What if it were effortless to fix a spelling error on the New York Times? Imagine contributing to the ReactJS documentation in an instant.
 
-With this workflow, we don't have to host a separate [Cloud Editing Environment](https://tinacms.org/blog/editing-on-the-cloud). The **live site is the editing environment**, with Git to keep the guide rails.
-
-## Crowd-sourcing your content
-
-We see this approach as _lowering the friction_ of website editing as a whole. Combine _Visual Open Authoring_ with [Inline Editing](https://tinacms.org/docs/inline-editing) and we have ourselves a digital-content revolution. What if it were effortless to fix a spelling error on the New York Times? Imagine contributing to the ReactJS documentation in an instant.
-
-**The web is all about community**, crowd-sourcing, and the power of many perspectives. The easier it is to make content contributions, the more viewpoints we incorporate. The less context switching, the _more space for creative expression_ of ideas.
+**The web is all about community**, crowdsourcing, and the power of many perspectives. The easier it is to make content contributions, the more viewpoints we incorporate.
```
Comment (kendallstrautman) [inline content/blog/introducing-visual-open-authoring.md]:
```suggestion
**The web is all about community**, crowdsourcing, and the power of many perspectives. The easier it is to make content contributions, the more viewpoints we incorporate. The less context switching, the _more space for creative expression_ of ideas.
```

Comment (kendallstrautman) [reply content/blog/introducing-visual-open-authoring.md]: I think this line about creativity is important

Comment (dwalkr) [reply content/blog/introducing-visual-open-authoring.md]: I removed that line for a reason. I'm not quite sure what it means. This conclusion really builds momentum, and that seemed to take it away

Comment (dwalkr) [reply content/blog/introducing-visual-open-authoring.md]: I think my problem is with its focus on "context switching". This paragraph hits very broad notes. Community, easy contribution, many perspectives, incorporating viewpoints, and.... context-switching?

Why does a lack of context-switching create "space"? What does VOA actually do to foster "space for creative expression"? I'm not denying that creativity is good, but does VOA really make an impact on this specific front?

Comment (kendallstrautman) [reply content/blog/introducing-visual-open-authoring.md]: I think the conclusion is trying to paint a bigger vision, with VOA in combination with the broader impact of Tina. Once of the biggest 'differences' with Tina is the aspect of the editing context. And given that it's mentioned in the introduction as what makes this version of OA different, it seems to be an important thread.

Comment (dwalkr) [reply content/blog/introducing-visual-open-authoring.md]: I guess you're right, the in-context aspect is an important part of the thesis. I'm reasonably convinced, back in it goes

**Pull request merged**